Library/Homebrew/rubocops/order_utils.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+def check_order(component_precedence_list, body_node)
```

Comment:
If these methods are going to be floating in a utils class they should be:
- `module_function` in a module
- have dedicated tests

Alternatively, they could be private methods in `components_order.rb` and there be two cops in that file.

Reply:
The `OnOsBlock` block test does more than testing order. It also checks the content of the on_macos/on_linux blocks. And I want to extend it for `depends_on` blocks that contain on_macos/on_linux urls. I'll update it with dedicated tests and add it to a module.